pipelines/batch/src/main/java/com/google/fhir/analytics/ParquetMerger.java

```diff
@@ -352,7 +351,7 @@ public static Pipeline writeMergedViewsPipeline(
       throws IOException {
     Pipeline pipeline = Pipeline.create(options);
     log.info("Merging materialized view {}", viewName);
-    ViewDefinition viewDef = viewManager.getViewDefinition(viewName);
+    ViewDefinition viewDef = viewManager != null ? viewManager.getViewDefinition(viewName) : null;
```

Comment:
Hmm, I think this is one of those cases that `viewManager` is guaranteed not to be `null` here (you can see it in the logic of the above function when this is called). So I would prefer that we change the signature and remove `@Nullable` from `viewManager`, then deal with it in the caller function above.